src/login/LoginFailure.jsx

```diff
@@ -68,6 +70,25 @@ const LoginFailureMessage = (props) => {
       );
       break;
     }
+    case ALLOWED_DOMAIN_LOGIN_ERROR: {
+      const url = `${getConfig().LMS_BASE_URL}/dashboard/?tpa_hint=${context.tpaHint}`;
+      const tpaLink = (
+        <a href={url}>
+          {context.provider} account
+        </a>
```

Comment:
"account" needs to be translated. 
Helpful reference: 
https://github.com/openedx/frontend-app-authn/blob/master/src/login/LoginFailure.jsx#L49-L52